soweego/importer/models/base_entity.py

```diff
@@ -35,3 +35,22 @@ class BaseEntity():
 
     def __repr__(self) -> str:
         return "<BaseEntity(catalog_id='{0}', name='{1}')>".format(self.catalog_id, self.name)
+
+
+class BaseRelationship():
+    __table_args__ = (
+        UniqueConstraint("catalog_id0", "catalog_id1"),
+    )
+
+    internal_id = Column(Integer, unique=True,
+                         primary_key=True, autoincrement=True)
+
+    catalog_id0 = Column(String(50), nullable=False, index=False)
```

Comment:
I suggest to give more eloquent names to these fields. Rationale: we have a relationship **from** a `catalog_id` **to** another `catalog_id`, so:
```suggestion
    from_catalog_id = Column(String(50), nullable=False, index=False)
```